.azuredevops/deploy-core-platform-infra.yaml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  - name: deployFromFeature
+    displayName: "Deploy from Feature Branch"
+    type: boolean
+    default: false
+
+trigger: none
+
+resources:
+  repositories:
+    - repository: ADPPipelineCommon
+      name: DEFRA/adp-pipeline-common
+      endpoint: DEFRA
+      type: github
+      ref: main
+
+extends:
+  template: /pipelines/common-infra-deploy.yaml@ADPPipelineCommon
+  parameters:
+    projectName: $(projectName)
+    deployFromFeature: ${{ parameters.deployFromFeature }}
+    environments:
```

Comment:
Environments are already defined in base template, we don't need to pass it explicitly from this pipeline.

Reply:
The default environment is only set to SND1, but I am deploying these resource in SND2 only.